### PR TITLE
Deactivate support for writing to ADIOS1 via the openPMD plugin

### PR DESCRIPTION
**Pull request by franzpoeschel**
See [here](https://github.com/ComputationalRadiationPhysics/picongpu/issues/3382#issuecomment-708935904) for a reasoning.

### CODE REVIEW

File: include/picongpu/plugins/openPMD/openPMDWriter.hpp [hunk 1]
```diff
@@ -140,6 +140,17 @@ namespace openPMD
             log< picLog::INPUT_OUTPUT >( "openPMD: open file: %1%" ) % fullName;
             openPMDSeries = std::unique_ptr<::openPMD::Series >(
                 new ::openPMD::Series( fullName, at, communicator, jsonConfig ) );
+            if( openPMDSeries->backend() == "MPI_ADIOS1" )
+            {
+                throw std::runtime_error(R"END(
```
Comment (sbastrakov) [inline include/picongpu/plugins/openPMD/openPMDWriter.hpp:145]: What does `R"END` mean?

Comment (franzpoeschel) [reply include/picongpu/plugins/openPMD/openPMDWriter.hpp:145]: It's C++11 syntax for multiline strings, see item 6 [here](https://en.cppreference.com/w/cpp/language/string_literal).

Comment (psychocoderHPC) [reply include/picongpu/plugins/openPMD/openPMDWriter.hpp:145]: @franzpoeschel  IS it possible to check in CMake that OpenPMD-api is at least build with ADIOS2 and/or HDF5?

Comment (franzpoeschel) [reply include/picongpu/plugins/openPMD/openPMDWriter.hpp:145]: Yep, that should be doable. Will get to it.

Comment (franzpoeschel) [reply include/picongpu/plugins/openPMD/openPMDWriter.hpp:145]: Recent commit does that. I left the runtime check in there.

**Review by sbastrakov — APPROVED** [review]

**Review by psychocoderHPC — APPROVED** [review]

**Comment by sbastrakov** [comment]
@franzpoeschel we just discussed in the office that we should now disable reading ADIOS1 checkpoints as well, so to completely remove support of ADIOS1. However it seems from the code, that the exception thrown with this PRs change already affects reading, not just writing? (I don't have an old checkpoint to test it readily).

**Comment by franzpoeschel** [comment]
Yep, that one should catch both reading and writing @sbastrakov. I think the fastest way to test would be by creating an ADIOS1 checkpoint. Quickly commenting out that check should do the trick.

**Pull request merged**